pkg/util/label/label.go

```diff
@@ -4,6 +4,15 @@ import (
 	"strings"
 )
 
+const (
+	OnepanelPrefix      = "onepanel.io/"
+	TagPrefix           = "tags.onepanel.io/"
+	WorkflowTemplate    = OnepanelPrefix + "workflow_template"
```

Comment:
For consistency, we should always use `-` instead of `_` for label keys and values.

Reply:
Done.